boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpurad_iron.dts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "nrf54h20dk_nrf54h20_cpurad.dts"
+
+/delete-node/&cpurad_rx_partitions;
+/delete-node/&cpuapp_rx_partitions;
+
+/* This is not yet an exhaustive memory map, and contain only a minimum required to boot
+ * the radio core.
```

Comment:
Seems a bit off given that this is the radio core, a slight rephrase might be needed.